recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+c_stdlib_version:   # [linux]
+  - 2.17            # [linux]
```

Comment:
Instead of doing this, would recommend adding the following to `conda-forge.yml`

```yaml
os_version:
  linux_64: cos7
  linux_aarch64: cos7
  linux_ppc64le: cos7
```

Then this can be dropped

```suggestion
```

Both are handled the same in conda-forge

Though find that `conda_build_config.yaml` files tend to accumulate more and more things once added, which inevitably cause conflicts with later migrations, etc. So prefer to avoid them if at all possible. In this case it is possible using `conda-forge.yml`, hence recommending that